AbstractTest.java

```diff
@@ -1,9 +1,7 @@
 abstract class AbTest {
-	public AbTest() {
-		System.out.println("inside AbTest constructor");

```

Comment:
hey jayram why you removed code add that one

Reply:
Manu , I added the code . please check approve my PR